### DISCUSSION TITLE
[홍제국] 과제 1번 제출합니다.

### DISCUSSION THREAD

**Discussion by ghost** [SKKUAutoLab/H-Mobility-Autonomous-Advanced-Course#20] [2024년도 과제 제출]
![Screenshot from 2024-07-09 23-51-26](https://github.com/SKKUAutoLab/H-Mobility-Autonomous-Advanced-Course/assets/175135744/c515fc90-6ba4-440e-947b-62163ff8be5a)

**Comment by Jinsun-Lee** [comment]
안녕하세요.
자율주행 통합실습 담당 조교입니다. 

아래 링크에서 설문조사를 완료해 주시길 바랍니다.
https://forms.gle/DDwJCEf9Zbs3fxFi8

감사합니다.

**Comment by Jinsun-Lee** [comment]
설문조사는 과제에 포함된 필수 항목입니다. 이를 확인하시고, 완료 바랍니다.

감사합니다.
